phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/containers/ContainersController.java

```diff
@@ -151,6 +163,11 @@ private void updateEngineSettings(RepositoryDTO repositoryDTO) {
                         .runLater(() -> new ErrorMessage(tr("Loading engine tools failed."), e, this.containersView)));
     }
 
+    private void updateVerbs(RepositoryDTO repositoryDTO) {
+        this.verbsManager.fetchAvailableVerbs(repositoryDTO,
+                verbs -> Platform.runLater(() -> this.verbs = verbs));
```

Comment:
Yep, JavaFX class should *never* be used outside Phoenicis-javafx

Reply:
I agree with @plata, we need to sometimes call `Platform.runLater` inside a callback passed to a manager class like `ApplicationsManager`, `EnginesManager` or `VerbsManager`, because of how the callbacks work.